### ISSUE TITLE
Load pretrained model

### ISSUE THREAD

**Issue by CuongNguyen218** [westerndigitalcorporation/YOLOv3-in-PyTorch#4]
Hi @wuhy08 ,
I want to load pretrain model only for backbone darknet 53. How can I do it. Beside that when i use your repo, I saw loss value is unstable and it so high. Can you provide me your train log of it.
